packages/components/psammead-bulleted-list/src/index.jsx

```diff
@@ -8,6 +8,8 @@ import {
 } from '@bbc/gel-foundations/spacings';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
 
+const BULLET_PADDING = '1.75rem';
```

Comment:
Just checking this value has been confirmed by UX?

Reply:
UX have confirmed it's ok 👍